lib/gemstash/upstream.rb

```diff
@@ -15,7 +15,7 @@ class Upstream
     def initialize(upstream, user_agent: nil)
       @uri = URI(CGI.unescape(upstream.to_s))
       @user_agent = user_agent
-      raise "URL '#{@uri}' is not valid!" unless @uri.to_s =~ URI::DEFAULT_PARSER.make_regexp
+      raise "URL '#{@uri}' is not valid!" unless @uri.to_s&.match?(URI::DEFAULT_PARSER.make_regexp)
```

Comment:
Why this is needed?

Reply:
I'll revert it. Its change is related to rubocop warnings.